packages/paste-design-tokens/tokens/global/background-color.yml

```
@@ -8,159 +8,159 @@ props:
   # base background
   color-background:
     value: "{!palette-gray-10}"
-    comment: Default background color for any container
+    comment: Background color used for containers.
   color-background-strong:
     value: "{!palette-gray-20}"
-    comment: Strong default background color
+    comment: Strong background color used for containers.
   color-background-stronger:
     value: "{!palette-gray-50}"
-    comment: Stronger default background color
+    comment: Stronger background color used for containers.
   color-background-strongest:
     value: "{!palette-gray-70}"
-    comment: Strongest default background color
+    comment: Strongest background color used for containers.
   color-background-inverse:
     value: "{!palette-gray-0-transparent-10}"
-    comment: Inverse background color for any container. Must be used on color-background-body-inverse.
+    comment: Inverse background color used for containers.
   color-background-inverse-strong:
     value: "{!palette-gray-0-transparent-20}"
-    comment: Strong inverse background color for any container. Must be used on color-background-body-inverse.
+    comment: Strong inverse background color used for containers.
 
   # brand backgrounds
   color-background-brand:
     value: "{!palette-blue-80}"
-    comment: Primary brand background, accessible with inverse text
+    comment: Background color used for brand.
   color-background-brand-strong:
     value: "{!palette-blue-90}"
-    comment: Strong primary brand background, accessible with inverse text
+    comment: Strong background color used for brand.
   color-background-brand-stronger:
     value: "{!palette-blue-100}"
-    comment: Stronger primary brand background, accessible with inverse text
+    comment: Stronger background color used for brand.
   color-background-brand-highlight:
     value: "{!amaranth}"
-    comment: Highlight brand background, accessible with black text
+    comment: Background color used for brand highlights.
   color-background-brand-highlight-weakest:
     value: "{!amaranth-transparent-10}"
-    comment: Weakest highlight brand background
+    comment: Weakest background color used for brand.
```

I updated the doc because it was `color-background-brand-weakest` there.

```suggestion
    comment: Weakest background color used for brand highlights.
```